apps/i18n/music/en_us.json

```diff
@@ -123,4 +123,6 @@
   "tuneKeyNotes": "Key Notes",
   "tuneAllNotes": "All Notes",
   "keyPad": "Key Pad: Press enter to edit, escape to exit",
-  "startOverMusicWorkspace": "This will clear all your work. This action cannot be reversed."}
+  "startOverMusicWorkspace": "This will clear all your work. This action cannot be reversed.",
+  "viewingOldVersion": "You are viewing an older version of this project. To continue working, you can either restore this version or go back to the current version."
```

Comment:
Thanks to @katiejofr and @dancodedotorg for help with this text.

Reply:
I think this can be hardcoded to be translated by Localize instead of CrowdIn, cc @wilkie can you confirm?